docs/getting-started-wsl.md

```diff
@@ -0,0 +1,48 @@
+# Software Setup on Windows Subsystem for Linux
+
+## Table of Contents
+
+* [Table Of Contents](#table-of-contents)
+* [Introduction](#introduction)
+* [WSL2 Setup](#wsl2-setup)
+* [X Server Setup](#wsl2-setup)
+
+
+## Introduction
+
+Windows has a new Windows Subsystem for Linux component that can be used to develop and run code for Linux on Windows. WSL1 was a Windows component that implemented Linux kernel interfaces, and didn't work great with Thunderbots code. WSL2 runs a full-fledged Linux kernel in a VM, and works great with Thunderbots and grSim with the exception that we need to use software rendering instead of GPU-accelerated rendering for grSim.
```

Comment:
Have you tried with our Visualizer? Does it face the same GPU limitations as grsim?

Reply:
Everything runs fine on my machine, but I have a relatively high spec (i7-9750h) machine. People with 2 cores are gonna have a very bad time

Reply:
If we deprecated the use of grsim, would we be able to leave the GPU-accelerated rendering enabled? (Just curious, since we haven't done this yet).

Reply:
see below, may be possible